### DISCUSSION TITLE
Update Unifi Network from 7.1.66 to 7.1.68

### DISCUSSION THREAD

**Discussion by Bolten88** [fabianishere/udm-iptv#155] [Q&A]
In the documentation i see this:

_**UniFi Dream Machine SE and UniFi Dream Router**
It is currently not possible to persist the installation across firmware updates (see #120). Your configuration should remain, so only re-installation is necessary._

Which steps do you need to again to make it work after an update? Or is it just the whole installation part?

**Download udm-iptv package**
curl -O -L https://github.com/fabianishere/udm-iptv/releases/download/v2.1.4/udm-iptv_2.1.4_all.deb
**Download a recent igmpproxy version**
curl -O -L http://ftp.debian.org/debian/pool/main/i/igmpproxy/igmpproxy_0.3-1_arm64.deb
**Update APT sources and install dialog package for interactive install**
apt update && apt install dialog
**Install udm-iptv and igmpproxy**
apt install ./igmpproxy_0.3-1_arm64.deb ./udm-iptv_2.1.4_all.deb

**Comment by bvk007** [comment]
Well actually. For a network upgrade you don't need to do a reinstall for udm-iptv. The network application is an application within your router. A firmware upgrade is an upgrade of Unifi-OS. So to keep things short. You can do an upgrade for network without losing your iptv config.

In case of a firmware upgrade, I know what my settings are (by hard ;) ). So I just do a ```dpkg-reconfigure -p medium udm-iptv```

But you could always keep a copy of /etc/udm-iptv.conf. And do a restore. Then restart your service with ```systemctl restart udm-iptv``` and you're good to go :)

**Reply by Bolten88** [reply]
That is good to hear. I will do the update tomorrow and will be more alert for the next UniFi-Os update. And i will save the configuration file just to be sure.

**Reply by Bolten88** [reply]
Have you updated to UniFi os version 2.4.15? And was it necessary to reinstall the iptv software?

**Reply by bvk007** [reply]
I do not have a UDM-SE or a UDR. I have a UDM-base. So my firmware versions are different than yours.
